src/core/include/openvino/op/roi_align.hpp

```diff
@@ -81,6 +81,88 @@ class OPENVINO_API ROIAlign : public Op {
     PoolingMode m_mode;
 };
 }  // namespace v3
+
+namespace v9 {
+class OPENVINO_API ROIAlign : public Op {
+public:
+    OPENVINO_OP("ROIAlign", "opset9", op::Op, 9);
+    BWDCMP_RTTI_DECLARATION;
```

Comment:
```suggestion
```

Reply:
Hi, @ilyachur : if remove this macro, it will throw error " error: ‘const ov::DiscreteTypeInfo ov::op::v9::ROIAlign::type_info’ is not a static data member of ‘class ov::op::v9::ROIAlign’
         const ov::DiscreteTypeInfo CLASS::type_info = CLASS::get_type_info_static(); "
do you also suggest to remove 'BWDCMP_RTTI_DEFINITION(op::v9::ROIAlign);' from 'src/core/src/op/roi_align.cpp' ?

Reply:
You need to remove both macros